templates/legal/index.html

```diff
@@ -23,78 +23,73 @@ <h1>Ubuntu legal</h1>
   <div class="row">
     <div class="u-equal-height">
       <div class="col-6 p-card">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/53640d35-picto-ubuntu-warmgrey.svg" alt="" />
-            <h3 class="p-heading-icon__title">
+        <h3>
           <a href="/legal/terms-and-policies">
             Terms and policies&nbsp;&rsaquo;
           </a>
         </h3>
-          </div>
-          <p>The main terms governing use of our software and websites, including links to privacy, intellectual property and copyright information.</p>
-        </div>
+        <p>The main terms governing use of our software and websites, including links to privacy, intellectual property and copyright information.</p>
       </div>
       <div class="col-6 p-card">
-        <div class="p-heading-icon">
-          <div class="p-heading-icon__header">
-            <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/03c5463a-picto-business-warmgrey.svg" alt="" />
-            <h3 class="p-heading-icon__title">
-              <a href="/legal/ubuntu-advantage">
-                Ubuntu Advantage terms&nbsp;&rsaquo;
-              </a>
-            </h3>
-          </div>
-          <p>A legal description of the services included in the Ubuntu Advantage support package.</p>
-        </div>
+        <h3>
+          <a href="/legal/data-protection">
```

Comment:
Copy doc says this links to a top level `/data-privacy ` page. I guess its wrong.

Reply:
There isn't a `data-privacy` page, just `data-protection` which we're awaiting copy for.

@frankiegranato Do you agree this should link to `/legal/data-protection`?